schemas/schemas/infoAssessment.json

```diff
@@ -97,6 +97,31 @@
             "description": "Whether the assessment will support group work.",
             "type": "boolean",
             "default": false
+        },
+        "groupMaxSize": {
+            "description": "Maximum size of a group.",
+            "type": "number",
+            "default": 5
```

Comment:
This doesn't seem to be true. It looks like the default is actually `null`? (And similarly for `groupMinSize`.)